src/Lanceur/Views/KeywordsViewModel.cs

```diff
@@ -309,8 +316,14 @@ private void SetupValidations(CompositeDisposable d)
         {
             var validateFileNameExists = this.WhenAnyValue(
                 x => x.SelectedAlias.FileName,
-                x => !x.IsNullOrEmpty()
-            );
+                x =>
+                {
+                    if (x.IsNullOrEmpty()) return false;
+                    if (!x.StartsWith('@')) return true;
+                    
+                    return x.StartsWith('@') && _macroLister.GetVisibleAttributes().Any(a => a.Name == x);
```

Comment:
Should be all macros. Include the visible one.
Create `GetAttributes` in the MacroLister